packages/blockchain-wallet-v4-frontend/src/scenes/DebitCard/index.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import { connect, ConnectedProps } from 'react-redux'
+
+import DebitCard from './template'
+
+class DebitCardContainer extends React.PureComponent<Props> {
```

Comment:
Could you make this a Functional Component?

Reply:
done